code/examples/channel/src/store.rs

```diff
@@ -113,6 +125,11 @@ impl Db {
             certificates.insert(height, encode_certificate(&decided_value.certificate)?)?;
         }
         tx.commit()?;
+        let values_bytes = decided_value.value.to_bytes()?.to_vec().len() as u64;
+        let certificates_bytes = encode_certificate(&decided_value.certificate)?.len() as u64;
```

Comment:
Let's try to not encode the certificate twice, and store the length of the encoded certificate above.

Reply:
Same for the decided value